src/main/java/dtu/dk/Controller/GameController.java

```diff
@@ -192,6 +201,40 @@ private void getInformation(boolean isHost) {
         ui.addTextToTextPane(username);
     }
 
+    public ArrayList<Pair<Peer, Player>> getActivePeers() {
+        return activePeers;
+    }
+}
+
+class DisconnectChecker implements Runnable{
+
+    ArrayList<Pair<Peer, Player>>  activePeerList;
+    public DisconnectChecker(GameController gameController){
+        activePeerList = gameController.getActivePeers();
+    }
+    public void run(){
+        int nextPeerIndex = 1;
+        while(true){
```

Comment:
I looks like the thread never dies. Would i be smart to kille this thread when the client is the only one left?